datadog/resource_datadog_integration_aws.go

```diff
@@ -83,6 +83,18 @@ func resourceDatadogIntegrationAws() *schema.Resource {
 				ConflictsWith: []string{"account_id", "role_name"},
 				Optional:      true,
 			},
+			"metrics_collection_enabled": {
+				Description: "Whether Datadog collects metrics for this AWS account.",
+				Type:        schema.TypeBool,
+				Default:     false,
+				Optional:    true,
+			},
+			"resource_collection_enabled": {
+				Type:        schema.TypeBool,
+				Description: "Whether Datadog collects a standard set of resources from your AWS account.",
+				Default:     false,
+				Optional:    true,
+			},
```

Comment:
```suggestion
			},
			"cspm_resource_collection_enabled": {
				Type:        schema.TypeBool,
				Description: "Whether Datadog collects cloud security posture management resources from your AWS account. This includes additional resources not covered under the general resource_collection.",
				Default:     false,
				Optional:    true,
			},
```

Could we also include the [`cspm_resource_collection_enabled`](https://docs.datadoghq.com/api/latest/aws-integration/#create-an-aws-integration) option in this PR?

Reply:
Sure I'll add that field too 😊